sprocs/sync_assessments.sql

```diff
@@ -269,39 +270,46 @@ BEGIN
                 title,
                 max_points,
                 number_choose,
-                best_questions
-            ) VALUES (
+                best_questions,
+                advance_score_perc
+            )
+            VALUES (
                 new_assessment_id,
                 (zone->>'number')::integer,
                 zone->>'title',
                 (zone->>'max_points')::double precision,
                 (zone->>'number_choose')::integer,
-                (zone->>'best_questions')::integer
+                (zone->>'best_questions')::integer,
+                (zone->>'advance_score_perc')::double precision
             )
             ON CONFLICT (number, assessment_id) DO UPDATE
             SET
                 title = EXCLUDED.title,
                 max_points = EXCLUDED.max_points,
                 number_choose = EXCLUDED.number_choose,
-                best_questions = EXCLUDED.best_questions
+                best_questions = EXCLUDED.best_questions,
+                advance_score_perc = EXCLUDED.advance_score_perc
             RETURNING id INTO new_zone_id;
 
             -- Insert each alternative group in this zone
             FOR alternative_group IN SELECT * FROM JSONB_ARRAY_ELEMENTS(valid_assessment.data->'alternativeGroups'->zone_index) LOOP
                 INSERT INTO alternative_groups (
                     number,
                     number_choose,
+                    advance_score_perc,
                     assessment_id,
                     zone_id
                 ) VALUES (
                     (alternative_group->>'number')::integer,
                     (alternative_group->>'number_choose')::integer,
+                    (alternative_group->>'advance_score_perc')::integer,
```

Comment:
I believe this should be casted as a `double precision`?

```suggestion
                    (alternative_group->>'advance_score_perc')::double precision,
```